packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -298,8 +298,8 @@ export class Meter implements api.Meter {
     });
 
     await Promise.all(metrics).then(records => {
-      records.forEach(metrics => {
-        metrics.forEach(metric => this._processor.process(metric));
+      records.forEach(record => {
```

Comment:
Nit: It sounds a bit weird - a singular form and then you have `forEach`. I would keep plural form maybe change name of variable in line 296 from `metrics` to `metricsRecords` etc. and then this would be not changed to preserve plural form WDYT ?.

Reply:
Sure, I had to double-check myself since I thought the nesting `forEach`es were odd.